packages/synapse-react-client/src/components/StorybookComponentWrapper.tsx

```diff
@@ -100,7 +100,10 @@ export function StorybookComponentWrapper(props: {
   const { storybookContext } = props
 
   useEffect(() => {
-    overrideEndpoint(storybookContext.globals.stack as SynapseStack)
+    overrideEndpoint(
+      (storybookContext.globals.stack as SynapseStack) ||
+        (storybookContext.parameters.stack as SynapseStack),
```

Comment:
Fallback to `parameters.stack` if `globals.stack` is null. We can set `parameters.stack` at the Component or Story level.